packages/flutter_tools/lib/src/commands/add_platform.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:yaml/yaml.dart';
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../cache.dart';
+import '../flutter_project_metadata.dart';
+import '../globals.dart' as globals;
+import '../reporting/reporting.dart';
+import '../runner/flutter_command.dart';
+import 'create_mixin.dart';
+
+class AddPlatformCommand extends FlutterCommand with CreateCommandMixin {
+  AddPlatformCommand() {
+    addPlatformsOptions();
+    addPubFlag();
+    addOfflineFlag();
+    addOverwriteFlag();
+    addOrgFlag();
+    addIOSLanguageFlag();
+    addAndroidLanguageFlag();
+  }
+
+  @override
+  final String name = 'add-platform';
+
+  @override
+  final String description = 'Command to add platforms to a existing flutter plugin project';
```

Comment:
"Command to " is redundant. See the existing flutter help text:

```
Available commands:
  analyze           Analyze the project's Dart code.
  assemble          Assemble and build flutter resources.
  attach            Attach to a running application.
  bash-completion   Output command line shell completion setup scripts.
  build             Flutter build commands.
  channel           List or switch flutter channels.
  clean             Delete the build/ and .dart_tool/ directories.
  config            Configure Flutter settings.
  create            Create a new Flutter project.
  devices           List all connected devices.
  doctor            Show information about the installed tooling.
  downgrade         Downgrade Flutter to the last active version for the current channel.
  drive             Runs Flutter Driver tests for the current project.
  emulators         List, launch and create emulators.
  format            Format one or more dart files.
  install           Install a Flutter app on an attached device.
  logs              Show log output for running Flutter apps.
  precache          Populates the Flutter tool's cache of binary artifacts.
  pub               Commands for managing Flutter packages.
  run               Run your Flutter app on an attached device.
  screenshot        Take a screenshot from a connected device.
  symbolize         Symbolize a stack trace from an AOT compiled flutter application.
  test              Run Flutter unit tests for the current project.
  upgrade           Upgrade your copy of Flutter.
  version           List or switch flutter versions.
```
The only ones that list commands are referring to subcommands.